python/nncase/__init__.py

```diff
@@ -212,6 +213,7 @@ def use_ptq(self, ptq_dataset_options: PTQTensorOptions) -> None:
             raise Exception("Unsupported Weights Quant Type")
 
         self._quantize_options.use_mix_quant = ptq_dataset_options.use_mix_quant
+        self._quantize_options.import_config_file = ptq_dataset_options.import_config_file
```

Comment:
It's better to name `quant_scheme` and store json content instead of file path. When nncase is intergrated to IDE or Web, transfering content is better than storing it in a file and passing the file path.

Reply:
What do you mean for quant_scheme? Just A json string? Is it too big as a parameter for real models? A mobilenetv2 json is about 2.4MB and more than 70000 lines.

Reply:
Passing 2.4MB string is not strange. Load or save the string from/to file is business logics that python/IDE/Web should consider. APIs should be flexible and simple.